src/mlagility/analysis/analysis.py

```diff
@@ -142,12 +142,7 @@ def call_benchit(
         build_state = build.load_state(
             cache_dir=tracer_args.cache_dir, build_name=build_name
         )
-        if build_state.info.opt_onnx_unsupported_ops:
-            model_info.status_message = "Unsupported op(s) " + ", ".join(
-                build_state.info.opt_onnx_unsupported_ops
-            )
-        else:
-            model_info.status_message = "Build Error: see log files for details."
```

Comment:
Stale vendor-specific code

Reply:
Is there still a way in which vendors can show custom messages directly (such as the Unsupported ops above) instead of having them buried in a log file?

Reply:
+1

Reply:
You can put any message you like in StageException and it will be displayed. For example, the ONNX op checker will already raise an exception that says "op X is not supported".